pom.xml

```diff
@@ -35,7 +35,7 @@
     <packaging>jar</packaging>
 
     <properties>
-        <version.elytron>1.1.0.Beta8</version.elytron>
+        <version.elytron>1.1.0.Beta9-SNAPSHOT</version.elytron>
```

Comment:
Do be careful with SNAPSHOT dependencies, at this point would prefer to tag more frequently and not let them get added in a commit.